webapp/lib/view.dart

```diff
@@ -39,8 +42,153 @@ DivElement projectTitle;
 
 class ContentView {
   DivElement contentElement;
+  DashboardView dashboardView;
 
   ContentView() {
-    contentElement = new DivElement();
+    contentElement = new DivElement()..classes.add('content');
+    dashboardView = new DashboardView();
+    dashboardView.activePackages.addAll(
+      [
+        new ActivePackagesViewPartial('Urgent conversations'),
+        new ActivePackagesViewPartial('Open conversations'),
+        new ActivePackagesViewPartial('Batch replies (Week 12)'),
+      ]);
+    dashboardView.availablepackages.addAll(
+      [
+        new AvailablePackagesViewPartial('Quick Poll',
+          'Ask a question with fixed answers',
+          ['Needs: Q/A, Labelling team, Safeguarding response', 'Produces: Dashboard for distribution of answers']),
+        new AvailablePackagesViewPartial('Information Service',
+          'Answer people\'s questions',
+          ['Needs: Response protocol, Labelling team, Safeguarding response', 'Produces: Thematic distribution, work rate tracker']),
+        new AvailablePackagesViewPartial('Bulk Message',
+          'Send set of people a once off message',
+          ['Needs: Definition of who. Safeguarding response', 'Produces: Success/Fail tracker'])
+      ]);
+
+      dashboardView.pupolateActivePackages();
+      dashboardView.pupolateAvailablePackages();
+      contentElement.append(dashboardView.dashboardViewElement);
+  }
+}
+
+class DashboardView {
+  List<ActivePackagesViewPartial> activePackages;
+  List<AvailablePackagesViewPartial> availablepackages;
+
+  DivElement dashboardViewElement;
+  DivElement _projectActionsContainer;
+  AnchorElement _conversationsLink;
+  AnchorElement _oversightDashboardLink;
+  DivElement activePacakgesContainer;
+  DivElement availablePackagesContainer;
+
+  DashboardView() {
+    activePackages = [];
+    availablepackages = [];
+    dashboardViewElement = new DivElement()..classes.add('dashboard');
+    _projectActionsContainer = new DivElement()
+      ..classes.add('dashboard__project-actions');
+    _conversationsLink = new AnchorElement()
+      ..classes.add('dashboard__project-actions-link')
+      ..text = "Go to conversations"
+      ..href = "#";
+    _oversightDashboardLink = new AnchorElement()
+      ..classes.add('dashboard__project-actions-link')
+      ..text = "Go to oversight dashboard"
+      ..href = "#";
+    activePacakgesContainer = new DivElement()
+      ..classes.add('active-packages');
+    availablePackagesContainer = new DivElement()
+      ..classes.add('available-packages');
+
+    _projectActionsContainer.append(_conversationsLink);
+    _projectActionsContainer.append(_oversightDashboardLink);
+    dashboardViewElement.append(_projectActionsContainer);
+    dashboardViewElement.append(activePacakgesContainer);
+    dashboardViewElement.append(availablePackagesContainer);
+  }
+
+  void pupolateActivePackages() {
+    activePacakgesContainer.children.clear();
```

Comment:
typo nit: should be `activePackagesContainer`

Reply:
Oops. Sorry for the spelling mistakes. Corrected.

Reply:
no worries, it happens!